src/tink/domspec/Attributes.hx

```diff
@@ -60,6 +60,7 @@ typedef TableCellAttr = {>GlobalAttr<Style>,
 
 typedef InputAttr = {>GlobalAttr<Style>,
   @:optional var checked(default, never):Bool;
+  @:html('checked') @:optional var defaultChecked(default, never):Bool;
```

Comment:
I think it should be:

```haxe
@:jsOnly @:optional var defaultChecked(default, never):Bool;
```

Reply:
Nah, this is correct (whereas I made an error with [`defaultSelected`](https://github.com/haxetink/tink_domspec/blob/ef076f28de4782fea0973d23ca3baee8d3739d0e/src/tink/domspec/Attributes.hx#L226)). From MDN:

<!--StartFragment-->
defaultChecked | Boolean:&nbsp;Returns / Sets&nbsp;the default state of a radio button or checkbox *as originally specified in HTML* that created this object.
-- | --
<!--EndFragment-->

With this information, coconut.html can determine that if you write `defaultChecked` it maps should map to `checked` in the generated HTML (although IIRC it doesn't do that yet).

As opposed to that `@:jsOnly` is meant for properties that exist only in the DOM APIs and have no corresponding HTML attributes - if there's even such a thing.